cli/testdata/coder_server_--help.golden

```diff
@@ -129,11 +129,8 @@ AI BRIDGE OPTIONS:
           The key to authenticate against the OpenAI API.
 
       --aibridge-rate-limit int, $CODER_AIBRIDGE_RATE_LIMIT (default: 0)
-          Maximum number of AI Bridge requests per rate window. Set to 0 to
-          disable rate limiting.
-
-      --aibridge-rate-window duration, $CODER_AIBRIDGE_RATE_WINDOW (default: 1m)
-          Duration of the rate limiting window for AI Bridge requests.
+          Maximum number of AI Bridge requests per minute. Set to 0 to disable
+          rate limiting.
```

Comment:
nit: sorry for being pedantic and not thinking before but maybe limit per second would be easier to use then per minute?